src/helpers/errors.js

```diff
@@ -7,10 +7,14 @@ export function handleErrors({ attributesErrors, errors }, setError) {
 
   if (attributesErrors && Object.keys(attributesErrors).length) {
     setError(
-      Object.keys(attributesErrors).map((fieldName) => ({
-        message: capitalize(attributesErrors[fieldName][0]),
-        name: fieldName,
-      }))
+      Object.keys(attributesErrors).map((fieldName) => {
+        return {
+          message: attributesErrors?.[fieldName].length
+            ? capitalize(attributesErrors?.[fieldName][0])
+            : '',
```

Comment:
![image](https://user-images.githubusercontent.com/15303963/84950210-ed53eb80-b0c4-11ea-9217-95a53ce36ed2.png)
this is caused because the api is sending the empty array for the `reset_password_token` "attribute" when it doesn't have errors, instead of not sending the key at all